<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ez4ms</class>
 <widget class="QWidget" name="ez4ms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ez4ms</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="tokensFormLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="accessTokenLabel">
       <property name="text">
        <string>Access Token</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="accessTokenLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="refreshTokenLabel">
       <property name="text">
        <string>Refresh Token</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="refreshTokenLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="formIdLabel">
       <property name="text">
        <string>Form ID</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="formIdLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="responseIdLabel">
       <property name="text">
        <string>Response ID</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="responseIdLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="receivedContentLabel">
       <property name="text">
        <string>Received Content</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="receivedContentTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="authenticateBtn">
       <property name="text">
        <string>Authenticate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveTokensBtn">
       <property name="text">
        <string>Save Tokens</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="requestContentBtn">
       <property name="text">
        <string>Request Content</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteContentBtn">
       <property name="text">
        <string>Delete Content</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createFormBtn">
       <property name="text">
        <string>Create Form</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateFormBtn">
       <property name="text">
        <string>Update Form</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addMCQBtn">
       <property name="text">
        <string>Add MCQ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="parseFileBtn">
       <property name="text">
        <string>Parse File</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
